ReactWindows/ReactNative.Shared/Views/Text/ReactSpanViewManager.cs

```diff
@@ -71,7 +72,27 @@ public void SetColor(Span view, uint? color)
         public void SetTextDecorationLine(Span view, string textDecorationLineValue)
         {
             var textDecorationLine = EnumHelpers.ParseNullable<TextDecorationLine>(textDecorationLineValue) ?? TextDecorationLine.None;
-
+#if WINDOWS_UWP
+            if (ApiInformation.IsApiContractPresent("Windows.Foundation.UniversalApiContract", 4))
+            {
+                switch (textDecorationLine)
+                {
+                    case TextDecorationLine.Underline:
+                        view.TextDecorations = TextDecorations.Underline;
```

Comment:
> view.TextDecorations = TextDecorations.Underline; [](start = 23, length = 50)

This fails to compile, unfortunately.
RNW targets RS1, and these APIs don't exist in that version.

IsApiContractPresent is helpful/used for the case where the version you target does have an API, but you want to check the presence since you support a [min, target] range that contains versions not supporting the API.

In this particular matter you'd need some "#if SDKVER >= X" construct that may or may not exist, not sure.
Anyway, even that's workable, the bigger issue is that the change offers nothing to those who follow the official RS1 target.

Perhaps we'll have to move this fix "elsewhere", unless a more "threshold friendly" solution is found.